lib/blacklight/search_builder_behavior.rb

```diff
@@ -1,5 +1,5 @@
 module Blacklight
-  class SearchBuilder
+  module SearchBuilderBehavior
```

Comment:
I'm not sure I understand why this shouldn't be a class. It sure looks like we took class inheritance and turned it into module includes instead.